frontend/src/components/Home.tsx

```diff
@@ -12,7 +12,7 @@ function Home() {
       <nav>
         <ResponsiveAppBar />
       </nav>
-      <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center' }}>
+      <div style={{ color: '#0f274a', display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
a pattern I've seen is to move these color values into a separate file so you can reference them in one place and then change across the entire site more easily...more of a TODO than a must do